tests/ui_tools/test_popups.py

```diff
@@ -755,7 +755,7 @@ def test_keypress_stream_members(self, mocker, key, widget_size):
                     "rendered_description": "hey",
                     "subscribers": [],
                     "date_created": None,
-                    "stream_weekly_traffic": 123,
```

Comment:
As per the commit description, if a stream is newly created (within a week), the `stream_weekly_traffic` would be `None` as per the API docs [here](https://zulip.com/api/get-subscriptions#response).

I was not sure if I should add new tests with this field as `None` or just change the pre-existing tests (to prevent adding unnecessary tests and keep the commit smaller). Let me know if adding new tests is better. :)

The latter did not break the tests because the tests check for popup height, which remains the same even if the field is `None` cause we're displaying an appropriate message in that case.

Reply:
I have added a new fixture with `stream_weekly_traffic` as `None`, without modifying the existing ones.